lms/djangoapps/instructor_task/tasks.py

```diff
@@ -166,3 +167,13 @@ def cohort_students(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('cohorted')
     task_fn = partial(cohort_students_and_upload, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+@task(base=BaseInstructorTask, routing_key=settings.GRADES_DOWNLOAD_ROUTING_KEY)  # pylint: disable=E1102
```

Comment:
Please use the pylint name, not the error code. I did a [huge cleanup](https://github.com/edx/edx-platform/pull/6090) to fix this :(